src/System.Memory/src/System/PinnableReferenceHelpers.Layouts.Microsoft.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Hence the difficulty in choosing a good name - with the "store as Pinnable" trick, though, we can mothball this file altogether and have a implementation that works on Mono too. So this is moot.